### PR TITLE
[Python 3.6] Solution to 6, 7 and new solution to 4

### PR DESCRIPTION
**Pull request by Atakyn**

### CODE REVIEW

File: challenge_6/Atakyn/c6.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# find the ranges in a list
+
+
+def ranges(array):
+    start = [array[0]]
```
Comment (zmiller91) [inline challenge_6/Atakyn/c6.py]: index out of bounds on empty array

File: challenge_6/Atakyn/c6.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    result = []
+    for s, e in zip(start, end):
+        if s != e:
+            result.append(f'{s}->{e}')
+
+    return result
+
+
+print(ranges([5, 6, 9, 10, 12, 13, 15, 16, 18, 19, 25, 30]))
```
Comment (zmiller91) [inline challenge_6/Atakyn/c6.py]: Add `print(ranges([]))`

**Comment by Atakyn** [comment]
Fixed @zmiller91

**Comment by Atakyn** [comment]
Also, in the challenge 6 README.md it states:

> Assume the input will not be empty

**Comment by zmiller91** [comment]
Ah, sorry.  I just always test the null and empty case, it's an easy case to test and it's usually easy to handle. 

- [x] **Code Correctness**
- [x] **Code Style**
- [x] **Code Documentation**
- [x] **Path Structure**
- [x] **Merge Pull Request**

Edit your PR title and add (ReadyToMerge) at the end.

**Pull request merged**